README.md

```diff
@@ -50,38 +58,56 @@ You don't have to worry about explicitly declaring the page's category.
 
 #### Controlling the order of a page in the category navigation sidebar
 
-Any pages within a category are automatically added to the category navigation sidebar, when a user is viewing that tab.
+Any pages within a category are automatically added to the category navigation
+sidebar, when a user is viewing that tab.
 
-[![Documentation Category Navigation Sidebar](readme_images/docs_category_navigation_screenshot.jpg)](#)
+[![Documentation Category Navigation Sidebar](images/readme/docs_category_navigation_screenshot.jpg)](#)
 
-The pages are sorted top-to-bottom using a custom `order` [front-matter variable](https://jekyllrb.com/docs/frontmatter/). You can control a page's order within the navigation sidebar by setting `order` to an appropriate integer value.
+The pages are sorted top-to-bottom using a custom `order`
+[front-matter variable](https://jekyllrb.com/docs/frontmatter/). You can
+control a page's order within the navigation sidebar by setting `order` to an
+appropriate integer value.
 
 Careful attention must be paid to nested pages.
 
 #### Nested pages
 
-Documentation pages can be nested by adding a new page to a category sub-directory, e.g., `_docs/quickstarts/subdir/sub_page.md`. This will cause your documentation page to be indented to the right.
+Documentation pages can be nested by adding a new page to a category
+sub-directory, e.g., `_docs/quickstarts/subdir/sub_page.md`. This will cause
+your documentation page to be indented to the right.
 
-You must be careful when `order`ing nested pages. As far as Jekyll is concerned, the hierarchy is flat for a given category. If `docs/quickstarts/top_page.md` has `order: 2` and `_docs/quickstarts/subdir/sub_page.md` has `order: 1`, then `sub_page` will appear **above** `top_page`.
+You must be careful when `order`ing nested pages. As far as Jekyll is
```

Comment:
Is "ordering" intentional?
![image](https://user-images.githubusercontent.com/6977544/27557579-fbee1b52-5a6e-11e7-97ed-c98e3608b2b3.png)